### ISSUE TITLE
Cues for disabled buttons besides color for hover icon

### ISSUE THREAD

**Issue by quarterback** [18F/cms-hitech-apd#1662]
Consider additional cues for disabled buttons besides color like a hover icon

<img width="270" alt="Screen Shot 2019-07-01 at 3 26 09 PM" src="https://user-images.githubusercontent.com/4461062/60470251-9daab480-9c14-11e9-850a-71cd04348072.png">

**Comment by nkkl** [comment]
Note that this is [not required by WCAG](https://www.w3.org/WAI/WCAG21/quickref/#contrast-minimum):

>The visual presentation of text and images of text has a contrast ratio of at least 4.5:1, except for the following: 
>
>[...]
>
>Incidental: Text or images of text that are **part of an inactive user interface component**, that are pure decoration, that are not visible to anyone, or that are part of a picture that contains significant other visual content, have no contrast requirement.

**Comment by mgwalker** [comment]
It might still be useful for, e.g., colorblind users?  Since we're using design system buttons, it might also make sense for us to ask them about it - fix once, fix everywhere!

**Comment by jeromeleecms** [comment]
Done - #1957 has been completed!
